dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java

```diff
@@ -95,14 +95,15 @@ public String formatDependencyPaths(Iterable<ClassPathEntry> entries) {
   }
 
   /**
-   * Returns mapping from the Maven coordinates to class path entries that are in the dependency
-   * tree.
+   * Returns mapping from the Maven coordinates to {@link ClassPathEntry}. The keys are the
+   * coordinates of the direct dependencies of the root nodes in {@link #dependencyPaths}. The
+   * values are all {@link ClassPathEntry}s in the subtree of the key.
    */
   public ImmutableSetMultimap<String, ClassPathEntry> coordinatesToClassPathEntry() {
```

Comment:
Why is this a multimap? Shouldn't each string coordinate such as com.google:foo:32.2 map to exactly one classpath entry?